UIkitDZ/ReaderViewController.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  ViewController.swift
+//  UIkitDZ
+//
+//  Created by Timofey Privalov on 30.06.2021.
+//
+
+import UIKit
+
+class ReaderViewController: UIViewController {
+    var text = UITextView()
+    var color = [UIColor.blue, UIColor.red, UIColor.orange, UIColor.brown, UIColor.green, UIColor.black, UIColor.white]
+    var colors = ["🟦", "🟥", "🟧", "🟫", "🟩", "⬛️", "⬜️"]
```

Comment:
```suggestion
    var colors = ["🟦", "🟥", "🟧", "🟫", "🟩", "⬛️", "⬜️"]
```можно объединить с первым или другое название